browser/resources/settings/brave_clear_browsing_data_dialog/brave_clear_browsing_data_on_exit_page.ts

```diff
@@ -3,85 +3,115 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck
-
-import {Polymer} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js';
-import {loadTimeData} from '../i18n_setup.js';
-import '../settings_shared.css.js';
-import '../settings_vars.css.js';
-import '../controls/settings_checkbox.js';
+import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js'
+import {BaseMixin} from '../base_mixin.js'
+import {PolymerElement} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
+import {loadTimeData} from '../i18n_setup.js'
+import '../settings_shared.css.js'
+import '../settings_vars.css.js'
+import '../controls/settings_checkbox.js'
 import {getTemplate} from './brave_clear_browsing_data_on_exit_page.html.js'
 
-Polymer({
-  is: 'settings-brave-clear-browsing-data-on-exit-page',
+export interface SettingsBraveClearBrowsingDataOnExitPageElement {
+  $: {
+    checkboxes: HTMLElement,
+  }
+}
```

Comment:
The `this.$.camelCase` accessor requires this interface.

Reply:
note for other reviewers - this is a standard pattern upstream